docs/source/_static/dpf_entry.html

```diff
@@ -0,0 +1,3407 @@
+                    <li><p class="my-main-menu" id="disp_exampleWorkflows" onclick="changeDisplay(this)">Workflow examples</p></li>
+                </ul>
+            </a>
+            <a class="my-main-menu" id="disp_howToUse" onmouseenter="changeDisplayMouseOver(this)" onmouseleave="changeDisplayMouseOut(this)">
+                How to use DPF's package
+                <ul class="content-list-topnav" style="display:none;">
+                    <li><p class="my-main-menu" id="disp_howToUseCPython" onclick="changeDisplay(this)">CPython</p></li>
+                    <li><p class="my-main-menu" id="disp_howToUseIPython" onclick="changeDisplay(this)">IPython</p></li>
+                </ul>
+            </a>
+        </div>
+        <div class="leftright">
+            <div class="sidenav" id="sidenav">
+                <h2>Available Operators</h2>
+                <input type="text" id="mySearch" onkeyup="myFunction()" placeholder="Search.." title="Type in a category">
+            </div>
+
+            <div id="displayer" class="main">
+            </div>
+        </div>
+    </div>
+    <div class="data" id="operator_list"><div class="main_text" id="mainText"><h1>Overview of Data Processing Framework</h1><div class="overview_des">The Data Processing Framework (<b>DPF</b>) is designed to provide numerical simulation users/engineers with a toolbox for accessing and transforming simulation data. DPF can access data from solver result files as well as several neutral formats (csv, hdf5, vtk, etc.). Various <i><b>operators</b></i> are available allowing the manipulation and the transformation of this data. DPF is a <b><i>workflow</i></b>-based framework which allows simple and/or complex evaluations by <b>chaining operators</b>. The data in DPF is defined based on physics agnostic mathematical quantities described in a self-sufficient entity called <i><b>field</b></i>. This allows DPF to be a modular and easy to use tool with a large range of capabilities. It's a product designed to handle large amount of data.</div><h2 class="h2-main"> Advantages </h2><dl class="dpf-advantages"><dt>Computation efficiency</dt>
+	<dd>DPF is a modern framework and it has been developed by taking advantages of new hardware architectures. Thanks to continued development, new capabilities are frequently added.</dd>
+<dt>Generecity</dt>
+	<dd>DPF is physic agnostic. Thus, its use is not limited to a particular field.</dd>
+<dt>Extensibility and Customization</dt>
+	<dd>DPF is developed around very few entities, one for the data (field) and one for the operation (operator). Each of DPF capability is developed through operators which allows for a very good componentization of the framework. DPF is also plugin based, this way, adding new features or handling new formats is fast and easy. With this componentization, thoses plugins and the usage of dpf scripting, the user can add his own capabilities and link his existing work with dpf.</dd>
```

Comment:
DPF is developed around two entities, one for......

Each DPF capability is developed through operators, allowing componentization of the framework.

 DPF is also plugin based so adding new features or handling new formats is fast and easy. With componentization, plugins, and DPF scripting, you can add your own capabilities and link your existing work with DPF.

Reply:
updated in TFS